tests/wpunit/AssetsTest.php

```diff
@@ -120,6 +120,47 @@ public function it_should_get_the_correct_url_when_wp_content_dir_and_wp_content
 		}
 	}
 
+	/**
+	 * @test
+	 *
+	 * @dataProvider constantProvider
+	 */
+	public function it_should_get_the_correct_url_when_wp_content_dir_and_wp_content_url_are_diff_and_assets_are_in_asset_group( $id, $constants ) {
+		$slugs = [
+			'fake1' => [ true, false ],
+			'fake2' => [ false, false ],
+			'fake3' => [ true, true ]
+		];
+
+		foreach ( array_keys( $slugs ) as $slug ) {
+			Assets::init()->remove( $slug . '-script' );
+			Assets::init()->remove( $slug . '-style' );
+		}
+
+		foreach ( $constants as $constant => $value ) {
+			$this->set_const_value( $constant, $value );
+			$this->assertEquals( $value, constant( $constant ) );
+		}
+
+		Config::reset();
+
+		Config::set_hook_prefix( 'bork' );
+		Config::set_version( '1.1.0' );
+		Config::set_path( constant( 'WP_PLUGIN_DIR' ) . '/assets' );
+		Config::set_relative_asset_path( 'tests/_data/' );
+		Config::add_group_path( 'fake-group-path', constant( 'WP_PLUGIN_DIR' ) . '/assets/tests', '_data/fake-feature' );
```

Comment:
This test uses a group path, but the group path is really an extension of the base one:
* base `constant( 'WP_PLUGIN_DIR' ) . '/assets'`
* group path `constant( 'WP_PLUGIN_DIR' ) . '/assets/tests'`

Can you write another test (do not modify this one that is exercising an edge case), where the group path is NOT an extension of the root path?